dotnet/targets/Xamarin.Shared.Sdk.props

```
@@ -229,7 +229,12 @@
 		<!-- Disable our own assembly IL stripping logic, because ILC does that already -->
 		<EnableAssemblyILStripping>false</EnableAssemblyILStripping>
 
-		<!-- We're using our own native main function when using NativeAOT -->
-		<CustomNativeMain>true</CustomNativeMain>
+		<!--
+			We're using our own native main function when using NativeAOT.
+			This is true for both: managed executables and app extensions that can be class libraries.
```

App extensions are always libraries:

```suggestion
			This is true for both: managed executables and app extensions (which are libraries).
```

---

Thanks, fixed.